src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js';
+
+const operate = (num1, num2, op) => {
+  const big1 = Big(num1);
+  const big2 = Big(num2);
+
+  let result;
+  switch (op) {
+    case '+':
+      result = big1.plus(big2);
+      break;
+    case '-':
+      result = big1.minus(big2);
+      break;
+    case 'x':
+      result = big1.times(big2);
+      break;
+    case '÷':
+      if (big2.toNumber() === 0) {
+        return 'Error alert!: unreasonable division by zero.';
+      }
+      result = big1.div(big2);
+      break;
+    default:
+      throw new Error(`Wrong operator ${op}`);
```

Comment:
Nice effort so far. :+1: You've put in a lot work into this.

- [ ] The `division`, `%` operators generate the error below. 
![error](https://user-images.githubusercontent.com/70150423/127862384-152672aa-2355-41dc-a107-9914db11357c.png)

- [ ]  The `X` operator does not function at all.